src/docs/release/breaking-changes/layout-builder-optimization.md

```diff
@@ -0,0 +1,170 @@
+---
+title: LayoutBuilder optimization
+description: LayoutBuilder and SliverLayoutBuilder call the builder function less often.
+---
+
+## Summary
+
+This guide explains how to migrate Flutter applications after [the LayoutBuilder
+optimization][1].
+
+## Context
+
+[LayoutBuilder][2] and [SliverLayoutBuilder][3] call the [builder][4] function
+more often than necessary to fulfil their primary goal of allowing apps to adapt
+their widget structure to parent layout constraints. This has led to less
+efficient and jankier applications because widgets are rebuilt unnecessarily.
+
+This transitively affects [OrientationBuilder][5] as well.
+
+In order to improve app performance the [LayoutBuilder optimization][1] was made,
+which results in the `builder` function to be called less often.
```

Comment:
=> which results in calling the `builder` function less often.

Reply:
Done.